.github/DISCUSSION_TEMPLATE/new-type.yml

```
@@ -0,0 +1,50 @@
+title: "Add: "
+labels: ["New type"]
+body:
+- type: input
+  id: name
+  attributes:
+    label: Type name
+    description: The name of the terms type you would like to add.
+    placeholder: Business Privacy Policy
+  validations:
+    required: true
+- type: textarea
+  id: alternative-type-names
+  attributes:
+    label: Alternative type names
+    description: Different names for the same terms type.
+    placeholder: Business Users Privacy Policy, Commercial Privacy Policy
+- type: input
+  id: writer
+  attributes:
+    label: Writer
+    description: The writer of the document, in most cases the service provider itself.
```

Always try to use synonyms rather than repeating the titles in questions, to maximise the chances that at least one of the ways to present the same data is understood 🙂 
```suggestion
    description: The author of the document, in most cases the service provider itself.
```